test/fixtures/transport/index.js

```diff
@@ -0,0 +1,12 @@
+'use strict'
+
+const fs = require('fs')
+const { once } = require('events')
+
+async function run (opts) {
+  const stream = fs.createWriteStream(opts.dest)
```

Comment:
@mcollina and I spoke on the phone, we'd like to improve transport definition further by allowing for auto line splitting and parsing for simpler transport implementations (every transport is solving the same problem, let's solve it here if possible). 

We discussed `for await of` but there's concerns that this serial processing can saturate worker thread

Reply:
https://github.com/pinojs/pino-abstract-transport should solve your requirements Dave.

Reply:
1. doesn't this hit the same issues
2. I'd rather have an standard API in core - this will also making documenting it much easier

Reply:
@mcollina bump :point_up:

Reply:
1. it does not have the same issue because the user can do as they like. They can use `for await`, they can use streams, essentially anything.
2. It has to be a separate module to avoid dependency cycles. Moving it to a separate package avoids the transports having a direct dependency on pino.

Reply:
Let's punt this to a future iteration (if at all) so we can get this landed